dashboard/app/models/script.rb

```diff
@@ -1472,6 +1472,8 @@ def get_assessment_script_levels
   def get_feedback_for_section(section)
     feedback = {}
     script_levels.each do |script_level|
+      # Filter down to only the levels a teacher can give feedback on
+      next unless ["Applab", "Gamelab", "Weblab"].include?(script_level.level.type)
```

Comment:
Just learned `script_level.level` is deprecated and shouldn't be used:

https://github.com/code-dot-org/code-dot-org/blob/4b11ec08899476dc7ee09be5c634d9ef01996c66/dashboard/app/models/script_level.rb#L66-L69

Reply:
what is the new way of doing this?